pkg/config/setup/config.go

```diff
@@ -241,7 +243,16 @@ var serverlessConfigComponents = []func(pkgconfigmodel.Setup){
 func init() {
 	osinit()
 	// Configure Datadog global configuration
-	datadog = pkgconfigmodel.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
```

Comment:
Could we add a comment explaining the different values for `DD_CONF_NODETREEMODEL ` and what the different config models do?

Reply:
Good call, done